core/designsystem/src/main/java/com/iteneum/designsystem/utils/TextUtils.kt

```diff
@@ -0,0 +1,29 @@
+package com.iteneum.designsystem.utils
+
+object TextUtils {
```

Comment:
Please add the documentation to this object at the top of the file like the author and date and description

Reply:
Ready!